proto/gds/models/v1/models.proto

```diff
@@ -8,29 +8,36 @@ option go_package = "github.com/trisacrypto/directory/pkg/gds/models/v1;models";
 // issuance and only public keys and certificate metadata should be exchanged between
 // directories.
 message CertificateRequest {
-    // ID only needs to be unique with the requesting directory service.
+    // A unique identifier generated by the directory service, should be a globally
+    // unique identifier generated by the replica specified in requesting_replica.
     string id = 1;
 
+    // The replica that generated the certificate request should be the only replica
+    // that handles the certificate request. The request is replicated for information
+    // and backup purposes only.
+    string requesting_replica = 2;
```

Comment:
Adding this here will require us to migrate the old CertificateRequest versions, but I think that's important so that only the requesting replica handles the certificate request.